Automattic/jetpack
# Social: Add account connection confirmation UI

This builds on #37196 and adds a component to allow the user to confirm the account to be connected. 

## Proposed changes:

* Add connection confirmation UI and logic

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Please ensure to sandbox D148435-code to ensure Facebook pages are correctly marked as already connected. 

* Goto Jetpack and Social admin pages
* Click on Add new connection
* Click on connect for any social network and complete the auth
* Confirm that you see the confirmation screen
* Confirm that the connection confirmation works as expected
* Confirm that marking a connection as Shared works as expected

https://github.com/Automattic/jetpack/assets/18226415/c62fd47c-8290-435b-bcbc-3e106daef58b
